controllers/users.js

```diff
@@ -232,6 +194,52 @@ var createUserSettings = function(data, response, callback) {
   });
 };
 
+var createPlace = function(data, response, callback) {
```

Comment:
This feels very similar to places.createPlaces. Maybe it should be in the Places controller and then see if there's some code we can reuse?